drivers/gpio/gpio_nrfx.c

```diff
@@ -324,7 +342,8 @@ static void cfg_level_pins(const struct device *port)
 	const struct gpio_nrfx_cfg *cfg = get_port_cfg(port);
 	uint32_t pin = 0U;
 	uint32_t bit = 1U << pin;
-	uint32_t level_pins = get_level_pins(port);
+	uint32_t level_pins = data->pin_int_en & (~data->trig_edge &
+						  ~data->double_edge);
```

Comment:
We could keep `get_level_pins` function. It makes the code more readable and ensures the implementation is identical.

Reply:
Okay, I've bringed it back.